app/views/directives/notifications/notification-body.html

```diff
@@ -1,7 +1,6 @@
 <div
   class="drawer-pf-notification-inner"
-  tabindex="0"
-  ng-click="$ctrl.customScope.markRead(notification)">
+  ng-click="notification.unread && $ctrl.customScope.markRead(notification)">
```

Comment:
extra commit in here from your other bug

Reply:
Crossed the streams...